react/modules/gaEvents.ts

```diff
@@ -20,16 +22,16 @@ import {
 } from './utils'
 import shouldMergeUAEvents from './utils/shouldMergeUAEvents'
 
-export function viewItem(eventData: PixelMessage['data']) {
+export function viewItem(eventData: ProductViewData) {
   if (!shouldMergeUAEvents()) return
 
   const eventName = 'view_item'
 
-  const { currency, product, list, position } = eventData
+  const { currency, product, list } = eventData
 
   const { selectedSku, productName, productId, categories, brand } = product
 
-  const { name: variant } = selectedSku
+  const { itemId: variant } = selectedSku
```

Comment:
I made this change here, and [in](https://github.com/vtex-apps/google-tag-manager/pull/112/files#diff-1ea8976bf29d0502073e0dd4b20555afe500d68ee35b13c5001f048facbff50dL87-R89) `selectItem` so we use the SKU's ID instead of name as the GA4's item `item_variant` property because that's what we were using in the UA's versions as well:

https://github.com/vtex-apps/google-tag-manager/blob/ab9d525e189f24d3821ba5a551f4d08681df1afd/react/modules/enhancedEcommerceEvents.ts#L82

https://github.com/vtex-apps/google-tag-manager/blob/ab9d525e189f24d3821ba5a551f4d08681df1afd/react/modules/enhancedEcommerceEvents.ts#L135

https://github.com/vtex-apps/google-tag-manager/blob/ab9d525e189f24d3821ba5a551f4d08681df1afd/react/modules/enhancedEcommerceEvents.ts#L361